peer/store-utils.go

```diff
@@ -79,12 +79,12 @@ func GetPeers() ([]Peer, error) {
 	if err != nil || pairs == nil {
 		return nil, err
 	}
-	// If there is only one entry in the store this indicates its a stand
-	// alone cluster, no need to return back it's own information
-	if len(pairs) == 1 {
+	if len(pairs) == 0 {
```

Comment:
@prashanthpai : As discussed, initMyUUID () is been called twice and that has caused this issue. I don't see that change reflected here. With this change, are you able to see both the peer details in get peers request? I doubt so since context.Restart is never set true which means self details are not added into the etcd store?